docs/rfcs/rfc-005-query-limits.md

```diff
@@ -25,7 +25,7 @@ Would result in the following type definitions being generated:
 ```graphql
 input MovieOptions {
   offset: Int
-  limit: Int! = 10
+  limit: Int! = 5
```

Comment:
From my discussion with him earlier, @danstarns did not end up outputting this in the schema - so shall we remove this or are you going to add it in @angrykoala?

Reply:
Agree, this is nice, but adding this means adding a lot of extra types
```suggestion
  limit: Int!
```